src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -84,6 +84,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('identity', help="Use assigned managed identity resource id or name if in the same resource group")
         c.argument('key_encryption_key', help="key vault key uri")
 
+    with self.argument_context('acr update', is_preview=True) as c:
+        c.argument('data_endpoint_enabled', get_three_state_flag(), help="enable dedicated data endpoint for client firewall configuration")
```

Comment:
```suggestion
        c.argument('data_endpoint_enabled', get_three_state_flag(), help="Enable dedicated data endpoint for client firewall configuration")
```

Reply:
CLI will format help automatically. But it is a good practice to write sentence with correct grammar.